core/templates/dev/head/pages/topic_editor/subtopics_editor/subtopics_list_tab_directive.html

```diff
@@ -1,169 +1,115 @@
-            <button type="submit" class="btn btn-success btn-sm">Save</button>
-            <button class="btn btn-default btn-sm" ng-click="closeSubtopicTitleEditor()">Cancel</button>
-          </form>
-        </span>
-      </div>
-      <div ng-if="isSubtopicEditorDisplayed(subtopic)">
-        <div ng-if="!subtopicEditorIsShown" class="preview">
-          <angular-html-bind html-data="htmlData">
-          </angular-html-bind>
-          <div class="pull-right">
-            <button type="button"
-                    class="btn btn-default save-button"
-                    ng-click="closePreviewSubtopicPage(htmlData)">
-              Edit
-            </button>
-          </div>
-        </div>
-        <div ng-if="subtopicEditorIsShown" class="oppia-editor-card-body">
-          <schema-based-editor schema="SUBTOPIC_PAGE_SCHEMA"
-                               local-value="editableHtmlData">
-          </schema-based-editor>
-
-          <div class="pull-right">
-            <button type="button"
-                    class="btn btn-default save-button"
-                    ng-click="openPreviewSubtopicPage(editableHtmlData)">
-              Preview
-            </button>
-            <button type="button"
-                    class="btn btn-success save-button"
-                    ng-disabled="!editableHtmlData"
-                    ng-click="updateHtmlData(subtopic.getId(), editableHtmlData)">
-              Save
-            </button>
-          </div>
+        <div style="max-height: 35vh; overflow-y: auto;">
+          <md-card class="skill-id"
+                   ng-repeat="skillDescription in subtopic.getSkillDescriptions() track by $index"
+                   dnd-draggable="skillDescription"
+                   dnd-dragstart="startMoveSkill(subtopicIndex, $index)"
+                   dnd-effect-allowed="move">
+          <a ng-href="<[getSkillEditorUrl(subtopicIndex, $index)]>" target="_blank"> <[skillDescription]> </a>
+          </md-card>
         </div>
-      </div>
+      </md-card>
+      <md-card class="subtopic new" ng-click="createSubtopic()">
+        <i class="material-icons">add</i>
+      </md-card>
     </div>
   </md-card>
-  <div ng-show="!subtopicDisplayed" class="help-text">
-    <h2> Click on a subtopic to show its editor</h2>
+
+  <md-card class="uncategorized-skill-ids"
+           dnd-list="uncategorizedSkillDescriptions"
+           dnd-drop="endMoveSkill(null)">
+    <h3> Uncategorized Skills </h3>
+    <div style="max-height: 45vh; overflow-y: auto;">
+      <md-card class="skill-id"
+               ng-repeat="skillDescription in uncategorizedSkillDescriptions track by $index"
+               dnd-draggable="skillDescription"
+               dnd-effect-allowed="move"
+               dnd-dragstart="startMoveSkill(null, $index)">
+        <a ng-href="<[getSkillEditorUrl(null, $index)]>" target="_blank"> <[skillDescription]> </a>
+      </md-card>
   </div>
```

Comment:
indent by 2

/cc @apb7 @kevinlee12 shouldn't we be catching this?

Reply:
Hi @seanlip! We do have a custom indent function which checks that the indentation of tags is a multiple of 2, in the same way as the HTML linter works. The linter, as well as this function, do not match the blocks.
The custom indent function also checks whether the attributes are aligned with the first leftmost attribute in case of multiple line tags.

Reply:
Hm, could the function be modified to check that the correct indentation of 2 is used? I see this sort of error quite often...

Reply:
I think we could do something regarding this. I will pick this up today and update this thread accordingly. Thanks!

Reply:
@seanlip: We can add another check for end tags. The indentation of end tags should match the indentation of start tags and at the same time, should be a multiple of 2. In that way, blocks will have the same indentation. Does this sound good?

Reply:
Done.